explorer/interpreter/resolve_names.cpp

```diff
@@ -176,6 +187,11 @@ auto NameResolver::AddExposedNames(const Declaration& declaration,
                                    StaticScope& enclosing_scope,
                                    bool allow_qualified_names)
     -> ErrorOr<Success> {
+  if (trace_stream_->is_enabled()) {
+    *trace_stream_ << "--- add exposed name `";
+    declaration.PrintID(trace_stream_->stream());
+    *trace_stream_ << "` (" << declaration.source_loc() << ")\n";
+  }
```

Comment:
I think we can skip this trace message. The message we get from actually introducing the name seems to be sufficient.